inc/form.class.php

```diff
@@ -1353,21 +1357,20 @@ public function duplicate() {
                }
 
                $new_target_change = new PluginFormcreatorTargetChange();
-               $new_target_change->add($update_target_change);
-               $new_target_item_id = $new_target_change->getID();
-               if (!$new_target_item_id) {
+               $update_target_change['title'] = $update_target_change['name'];
+               if (!$new_target_change->update($update_target_change)) {
                   return false;
                }
                break;
          }
 
-         $target->update([
-            'id'        => $target->getID(),
-            'items_id'  => $new_target_item_id,
-         ]);
-
          switch ($target_values['itemtype']) {
             case PluginFormcreatorTargetTicket::class:
+               // Drop default actors
+               $target_ticket_actor->deleteByCriteria([
+                  'plugin_formcreator_targettickets_id' => $new_target_item_id
+               ]);
+
```

Comment:
whe nthe target ticket is created there are default actors added automatically. We need to drop them and create those from the source target ticket